helper/general_objects/schemas.go

```diff
@@ -189,6 +189,130 @@ var PageableSchema = map[string]*schema.Schema{
 	},
 }
 
+var geoPointFieldsDefSchema = map[string]*schema.Schema{
```

Comment:
not sure if it is a good idea to factorize geoPointFieldsDefSchema &  geoPointFieldsSchema (they differ like this: for geopoints used in DefinitionAttributes, the different fields have a defaultValue attribute, whereas the fields of the geopoints used in ValueAttributes have a value attribute)